onboarding/sprint-rotation.md

```diff
@@ -5,49 +5,43 @@ description: What is the sprint rotation and how does it work?
-
+Once an engineer has gone through general onboarding and done some basic set-up, they'll spend the following few
+sprints rotating through a variety of engineering teams.
+
+### Why rotate?
+
+We want engineers to be familiar with the full breadth of products, systems, and people at Artsy, but at a
+comfortable pace. Rather than immediately joining a team for the long-term and focusing on that team's priorities,
+rotations allow engineers to tour the teams, get to know other engineers, and touch a variety of layers and
+systems. These rotations may influence the choice of team an engineer joins, or may just provide valuable context
+to their future work.
+
+### How do sprint rotations work?
+
+Managers will arrange for new engineers to join a sequence of 3 or more teams for 1 sprint each. Emphasis should be
+on encountering a variety of teams and technology rather than having impact in a familiar space. Following these
+rotations, the engineer will join a team on a more stable basis.
+
+#### Managers' responsibilities
+
+- Taking into account other rotating engineers and individual teams' schedules, managers decide an appropriate
+  sequence of rotations and add each to the
+  [engineering onboarding calendar](https://calendar.google.com/calendar/b/1?cid=YXJ0c3ltYWlsLmNvbV9jc2RocTFnNHNyMmhhY2Nvc3RmMTA1bTEwc0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t)🔒,
+  inviting the engineer as well as the chosen team's technical lead and product manager.
+- For each rotation, managers should nominate a "buddy" from the team. This other engineer can help the new
+  engineer get set up, pair on their own sprint work, and be a primary point of contact for context about tickets
+  and projects. A new hire's [mentor](/onboarding/mentors.md) is a great choice for initial rotation "buddy."
+- Collaborate with each team's technical lead to ensure a healthy backlog of
+  [`good-first-issue`s](https://artsyproduct.atlassian.net/issues/?jql=labels%20%3D%20good-first-issue).
+
+#### New engineers' responsibilities
+
+- Engineers should join team slack channels, distribution lists, and events. (Confirm that joining the team's
+  google group adds team events such as sprint planning to your calendar.)
+- When possible, pair with your buddy or other team members on their sprint work.
+- When a pair isn't available, it can be useful to work independently on:
+  - setting up development environments for the team's major systems
+  - reviewing the [team's documentation](https://www.notion.so/artsy/Product-470238180cf94c87906ef1d3ee259e05)🔒
+  - [`good-first-issue` tickets](https://artsyproduct.atlassian.net/issues/?jql=labels%20%3D%20good-first-issue)
+    within the team's backlog
+- Schedule a convenient time with the product manager to be introduced to the team's goals and metrics.
```

Comment:
I'd be specific here-- `Schedule a convenient time (30 min should be sufficient) with the...`